<?xml version="1.0"?>
<launch>
	<arg name="topic" default="/soma/vel" doc="publish topic name" />
	<arg name="publish_rate" default="1.0" />
	<!--  -->
	<arg name="server_ip" default="192.168.1.79" />
	<arg name="send_port" default="22346" />
	<arg name="recv_port" default="12346" />
	<node name="atv_rotary_comm" pkg="soma_sensor" type="atv_rotary_communication_node">
		<remap from="/vel" to="$(arg topic)" />
		<param name="publish_rate" value="$(arg publish_rate)" />
		<param name="server_ip" value="$(arg server_ip)" />
		<param name="send_port" value="$(arg send_port)" />
		<param name="recv_port" value="$(arg recv_port)" />
	</node>
</launch>
